1-js/06-advanced-functions/09-call-apply-decorators/01-spy-decorator/task.md

```diff
@@ -27,4 +27,4 @@ for (let args of work.calls) {
 }
 ```
 
-P.S. That decorator is sometimes useful for unit-testing. Its advanced form is `sinon.spy` in [Sinon.JS](http://sinonjs.org/) library.
+Pós-escrito: Este decorador é algumas vezes útil para testes unitários. Sua forma avançada é `sinon.spy` na biblioteca [Sinon.JS](http://sinonjs.org).
```

Comment:
```suggestion
P.S. Esse decorador pode eventualmente ser útil para testes unitários. Sua forma avançada é `sinon.spy` na biblioteca [Sinon.JS](http://sinonjs.org).
```

Reply:
Fique a vontade para manter o "Pós-escrito" :)

Reply:
@Peruibeloko, I need to keep it because it helps increase the comprehension of the content.